content/blog/2019-12-compat.md

```diff
@@ -0,0 +1,190 @@
++++
+date = "2019-12-11"
+title = "Announcing Tokio-Compat"
+description = "December 11, 2019"
+menu = "blog"
+weight = 984
++++
+
+The [release][release-02] of Tokio 0.2 was the culmination of a great deal of
+hard work from numerous contributors, and has brought several significant
+improvements to Tokio. Using `std::future` and async/await makes writing async
+code using Tokio [much more ergonomic][await], and a new scheduler
+implementation makes Tokio 0.2's thread pool [as much as 10x faster][10x].
+However, updating existing Tokio 0.1 projects to use 0.2 and `std::future` poses
+some new challenges. Therefore, we're very excited to announce the release of
+the `tokio-compat` crate to help ease this transition.
```

Comment:
TIOLI: looking at this paragraph and the one below, I wonder if it might be better to pull some content out of the Motivation section and add in a few sentences of conclusion text, so that readers who stop reading the blog post after the first section (many of them?) understand a little of what the `tokio-compat` crate does.